src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,106 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+@SuppressWarnings("unchecked")
 public class ArrayList<T> implements List<T> {
+
+    private static final int DEFAULT_CAPACITY = 10;
+    private T[] elements;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```

it will be initialized by 0 by default